src/app/common/DeleteResourceModal.vue

```diff
@@ -6,21 +6,39 @@
     :modal-id="props.modalId"
     :title="props.title"
     type="danger"
-    data-testid="delete-resource-modal"
+    :data-testid="props.modalId"
```

Comment:
What is `:modal-id` used for?

Reply:
Not totally sure if we ever used `modalId` on KPrompt (it’s set a few lines above). Perhaps we don’t need this. Let me check.

Reply:
Sorry yeah thats kinda the one I was referring to a few lines up. I was wondering why we can't just pass `data-testid` in from above, rather than have a `props.modalId` but I wasn't sure why the KPrompt needed `:modal-id`

Reply:
I’ve removed this and just set `data-testid` as usual.